package.json

```diff
@@ -28,6 +28,7 @@
     "eslint-loader": "^4.0.2",
     "ipfs-http-client": "^57.0.3",
     "it-all": "^1.0.6",
+    "jose": "^4.8.3",
```

Comment:
why we are using this package???

Reply:
It's necessary to decode the credentials and get the account information because the response is only a token